risc0/zkvm/src/receipt.rs

```diff
@@ -298,6 +303,16 @@ pub struct SegmentReceipt {
 }
 
 /// Context available to the verification process.
+#[cfg(not(feature = "disable-dev-mode"))]
+pub struct VerifierContext {
+    /// A registry of hash functions to be used by the verification process.
+    pub suites: BTreeMap<String, HashSuite<BabyBear>>,
+    /// DevMode facilitates development by generating a fake receipt
```

Comment:
Let's expand the docs here as well. Perhaps something like the below. I did notice I duplicated a fair amount of copy, if someone has an alternative with less duplication I'm open to that as well.
```suggestion
    /// DevMode facilitates development by generating a fake receipt
    ///
    /// DevMode facilitates development by generating a fake receipt
    ///
    /// When this is `true`, the verifier DOES NOT CHECK if the cryptographic
    /// information in the receipts is correct. This enables faster development
    /// by bypassing the need for honest receipts, but it provide NO SECURITY
    /// WHATSOEVER. Do not use in production.
    ///
    /// DevMode is only used when the environment variable `DEV_MODE` is set. It
    /// can also be fully disabled at compile time, regardless of environment
    /// variables, by setting the feature flag `disable-dev-mode` on the
    /// `risc0_zkvm` crate.
```

Reply:
I'm not sure we need this bool. In my mind, verify would just check to see if the `InnerReceipt` was `Fake`.

Reply:
yeah, you're right. I realized that I made this more complicated than necessary. We can just check the env var because we aren't supporting dev mode in `no_std` anyway.